### ISSUE TITLE
Windows Mapped Drive Problems (recent windows 11 update?)

### ISSUE THREAD

**Issue by Nick-Doe** [Bouni/kicad-jlcpcb-tools#486]
The problems I was having with assigning parts (issue #474) seems to be related mapped drives after a recent windows update.
I have my work files mapped to  drive I.
If I open the kicad files using 
I:\...\file.kicad_pro
then I can't assign parts or generate a correct BOM
Using C:\I\...\file.kicad_pro
everything is OK 

(assigning parts seems a lot slower?)

**Comment by Bouni** [comment]
I'll see if I'm able to reproduce this issue locally

**Comment by Bouni** [comment]
I cannot reproduce the behaviour you describe ...

1. I created a new project on a sahred drive (Synology NAS)
2. I Added two parts to the schematic and assigned a LCSC Number to one of them
3. I switched to the PCB editor and placed both parts
4. I opend the plugin, the alrteday assigned part shows up correct in the table
5. I assigned a number to the second part via the Assign LCSC number button
6. Again the table is correct
7. I generated the production files, BOM and CPL are both valid

Is there a difference in this workflow to whta you do?
